### PR TITLE
Update getting started docs

### PR DESCRIPTION
**Pull request by paulczar**
addresses #1154

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

### CODE REVIEW

**Comment by aanand** [comment]
Thanks!

I agree with your comment on #1154 that this doesn't really solve that issue, but it might well make the tutorial flow better in any case. I like the clarification at the end about what you should actually expect to see as well.

Two things:

- We wrap lines in the docs at 80 characters.
- The last commit is missing a signoff. Not a huge issue for docs changes, but it'd be nice to see a green build.

@fredlf @SvenDowideit - thoughts?

File: docs/index.md [hunk 1]
```diff
@@ -134,19 +117,21 @@ Next, define a set of services using `docker-compose.yml`:
...
+
+* Builds from the `Dockerfile` in the current directory. 
+* Defines to run the command `python app.py` inside the image on start.
+* Forwards the exposed port 5000 on the container to port 5000 on the host machine.
+* Connects the web container to the Redis service via a link. 
+* Mounts the current directory on the host to `/code` inside the container allowing you to modify the code without having to rebuild the image.
+
+#### redis
+
+* Uses the public [redis](https://registry.hub.docker.com/_/redis/) image which gets pulled from the Docker Hub registry.
```
Comment (fredlf) [inline docs/index.md]: Redis

File: docs/index.md [hunk 2]
```diff
@@ -157,7 +142,12 @@ image for your code, and start everything up:
     web_1   |  * Running on http://0.0.0.0:5000/
 
 The web app should now be listening on port 5000 on your Docker daemon host (if
-you're using Boot2docker, `boot2docker ip` will tell you its address).
+you're using Boot2docker, `boot2docker ip` will tell you its address). In a browser,
+open `http://ip-from-boot2docker:5000` and you should get a message in your browser saying:
+
+`Hello World! I have been seen 1 times.`
+
+Refreshing the page will see the number increment.
```
Comment (fredlf) [inline docs/index.md]: Refreshing the page will increment the number.

**Comment by fredlf** [comment]
+1 to this change. I think it clarifies the tutorial flow.

**Comment by chanezon** [comment]
This also fixes #1148 
But the format of the yml file should be shown at the beginning.
I will send a proposal.

**Comment by chanezon** [comment]
Done at https://github.com/docker/compose/pull/1200

**Comment by aanand** [comment]
Superseded by #1200. Thanks, @paulczar!

**Pull request closed without merging**